composer.json

```diff
@@ -13,15 +13,15 @@
     ],
     "require": {
         "php": "^5.3.9|^7.0",
-        "symfony/framework-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.3|~3.0"
     },
     "require-dev": {
         "twig/twig": "^1.12|^2.0",
         "symfony-cmf/testing": "^1.3",
         "doctrine/phpcr-odm": "^1.3",
         "doctrine/annotations": "^1.1.2",
         "jms/serializer-bundle": "^0.12|^1.0",
-        "liip/imagine-bundle": "~1.0.4",
+        "liip/imagine-bundle": "~1.5.3",
```

Comment:
are we no longer compatible with imagine-bundle 1.0.*? if its to allow a version that supports symfony 3, lets change the line to `^1.0.4`